src/types/sdk/services/transactions.ts

```diff
@@ -0,0 +1,16 @@
+import type { RequestOptions } from '@/types/sdk';
+
+type CreateTransactionResponse = {
+  id: string;
+  createdBy: string;
+  createdDate: string;
```

Comment:
This is returned from the api as `created_at`, which would need to translate into `createdAt` with the camel case conversion here.

Reply:
nice catch